Sming/Components/libsodium/Readme.md

```diff
@@ -0,0 +1,16 @@
+# libsodium
```

Comment:
Please rename file to `README.md` so doc. system picks it up.

Reply:
done